long-ride-alerts/README.md

```diff
@@ -77,7 +77,7 @@ The challenge is figuring out what state and timers to use, and when to set and
 ## Documentation
 
 - [ProcessFunction](https://ci.apache.org/projects/flink/flink-docs-stable/dev/stream/operators/process_function.html)
-- [Working with State](https://ci.apache.org/projects/flink/flink-docs-stable/dev/stream/state/index.html)
+- [Working with State](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/fault-tolerance/state)
```

Comment:
This link should point to flink-docs-stable, not master

Reply:
Fixed